src/startup.ts

```diff
@@ -2,24 +2,33 @@ import * as events from "./helpers/events";
 import * as time from "./statistics/time";
 import * as window from "./window/window";
 import * as windowWarning from "./window/windowWarning";
-
+import { StatController } from "./objects/StatController";
+import { timeSpentStatistic } from "./statistics/timeSpent";
 
 /**
  * The entry point for this plugin. Should initialize any tracking of statistics, and if the ui
  * is enabled, it should also initialize the windows.
+ *
+ * To add new widgets, you can create a new Statistic object and add it to the StatController.
  */
-export function startup()
-{
-	events.initialize();
-	time.initialize();
+export function startup() {
+  // a container to hold all the statistics data/widgets for easy pausing and resetting
+  const statController = new StatController();
+  events.initialize();
+  time.initialize();
```

Comment:
Missed this last time. This is the legacy time instance. It shouldn't be imported nor initialized (honestly, the legacy statistic and window can probably be deleted, as if they need to be referenced, they are [in the v1.0.0 tag](https://github.com/KatieZeldaKat/openrct2-statistics/tree/v1.0.0)).